Problem running transloco-keys-manager find on Windows

<!--
PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.

ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
-->

## I'm submitting a...

<!-- Check one of the following options with "x" -->
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

After installing tranloco and transloco key manager tried running: npm run i18n:find and got:
<pre>
√ Extracting Template and Component Keys �
- Checking for missing keys ✨M:\Works\agro\agro-client\node_modules\@ngneat\transloco-keys-manager\helpers\getScopeAndLangFromFullPath.js:1
"use strict";function getScopeAndLangFromFullPath(e,t){!1===t.endsWith("/")&&(t=`${t}/`);const[n,o]=e.split(t),l=o.split("/");let p,r;return l.length>1?(r=l.pop().replace(".json",""),p=l.join("/")):r=l[0].replace(".json",""),{scope:p,lang:r}}Object.defineProperty(exports,"__esModule",{value:!0}),exports.getScopeAndLangFromFullPath=getScopeAndLangFromFullPath;
                                                                                                                   ^

TypeError: Cannot read property 'split' of undefined
    at Object.getScopeAndLangFromFullPath (M:\Works\agro\agro-client\node_modules\←[4m@ngneat←[24m\transloco-keys-manager\helpers\getScopeAndLangFromFullPath.js:1:116)
    at Object.compareKeysToFiles (M:\Works\agro\agro-client\node_modules\←[4m@ngneat←[24m\transloco-keys-manager\keysDetective\compareKeysToFiles.js:1:1024)
    at Object.findMissingKeys (M:\Works\agro\agro-client\node_modules\←[4m@ngneat←[24m\transloco-keys-manager\keysDetective.js:1:835)
    at Object.<anonymous> (M:\Works\agro\agro-client\node_modules\←[4m@ngneat←[24m\transloco-keys-manager\index.js:2:744)
←[90m    at Module._compile (internal/modules/cjs/loader.js:959:30)←[39m
←[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)←[39m
←[90m    at Module.load (internal/modules/cjs/loader.js:815:32)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:727:14)←[39m
←[90m    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)←[39m
←[90m    at internal/main/run_main_module.js:17:11←[39m
</pre>

## Expected behavior

Lovely list that summarizes the keys found

## Minimal reproduction of the problem with instructions

Executed:
ng add @ngneat/transloco
ng g @ngneat/transloco:keys-manager

Which generated scripts in package.json:

"i18n:extract": "transloco-keys-manager extract",
"i18n:find": "transloco-keys-manager find"

Executed:
npm run i18n:find

Tried to change transloco.config.js: keysManager.translationsPath to some other variants like: with /without ending slash, or with backslashes

## Environment

<pre><code>
Angular version: 8.2.12

For Tooling issues:
- Node version: v12.13.1
- Platform:  Windows 7 Ultimate

Others:
transloco.config.js:
module.exports = {
	rootTranslationsPath: 'src/assets/i18n/',
	langs: ['en', 'lv'],
	keysManager: {
		input: 'src/app',
		translationsPath: 'src/assets/i18n',
		sort: true,
		addMissingKeys: true,
		replace: true
	}
};

</code></pre>

For me it happens as easy as:

ng new transloco1
	routing - No
	CSS

cd transloco1

ng add @ngneat/transloco
	en, lv
	SSR - No
	
ng g @ngneat/transloco:keys-manager
	Both

npm run i18n:find

and result is:

```bash
M:\Works\example\transloco1>npm run i18n:find

> transloco1@0.0.0 i18n:find M:\Works\example\transloco1
> transloco-keys-manager find


 � � Staring Search For Missing Keys � �

√ Extracting Template and Component Keys �
- Checking for missing keys ✨M:\Works\example\transloco1\node_modules\@ngneat\transloco-keys-manager\helpers\getScopeAndLangFromFullPath.js:1
"use strict";function getScopeAndLangFromFullPath(e,t){!1===t.endsWith("/")&&(t=`${t}/`);const[n,o]=e.split(t),l=o.split("/");let p,r;return l.length>1?(r=l.pop().replace(".json",""),p=l.join("/")):r=l[0].replace(".json",""),{scope:p,lang:r}}Object.defineProperty(exports,"__esModule",{value:!0}),exports.getScopeAndLangFromFullPath=getScopeAndLangFromFullPath;
                                                                                                                   ^

TypeError: Cannot read property 'split' of undefined
    at Object.getScopeAndLangFromFullPath (M:\Works\example\transloco1\node_modules\←[4m@ngneat←[24m\transloco-keys-manager\helpers\getScopeAndLangFromFullPath.js:1:116)
    at Object.compareKeysToFiles (M:\Works\example\transloco1\node_modules\←[4m@ngneat←[24m\transloco-keys-manager\keysDetective\compareKeysToFiles.js:1:1024)
    at Object.findMissingKeys (M:\Works\example\transloco1\node_modules\←[4m@ngneat←[24m\transloco-keys-manager\keysDetective.js:1:835)
    at Object.<anonymous> (M:\Works\example\transloco1\node_modules\←[4m@ngneat←[24m\transloco-keys-manager\index.js:2:744)
←[90m    at Module._compile (internal/modules/cjs/loader.js:959:30)←[39m
←[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)←[39m
←[90m    at Module.load (internal/modules/cjs/loader.js:815:32)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:727:14)←[39m
←[90m    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)←[39m
←[90m    at internal/main/run_main_module.js:17:11←[39m
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! transloco1@0.0.0 i18n:find: `transloco-keys-manager find`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the transloco1@0.0.0 i18n:find script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\MME\AppData\Roaming\npm-cache\_logs\2019-11-19T19_22_24_764Z-debug.log
```

Tested on different Windows 7 Professional PC with newest Angular-CLI and on C: drive :
```bash
Angular CLI: 8.3.19
Node: 10.15.1
OS: win32 x64
Angular: 8.2.14
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.19
@angular-devkit/build-angular     0.803.19
@angular-devkit/build-optimizer   0.803.19
@angular-devkit/build-webpack     0.803.19
@angular-devkit/core              8.3.19
@angular-devkit/schematics        8.3.19
@angular/cli                      8.3.19
@ngtools/webpack                  8.3.19
@schematics/angular               8.3.19
@schematics/update                0.803.19
rxjs                              6.4.0
typescript                        3.5.3
webpack                           4.39.2
```
Result is the same:
```bash
C:\Users\Arnis\Job\test\transloco1>npm run i18n:find

> transloco1@0.0.0 i18n:find C:\Users\Arnis\Job\test\transloco1
> transloco-keys-manager find


 ? ? Staring Search For Missing Keys ? ?

? Extracting Template and Component Keys ?
- Checking for missing keys ?C:\Users\Arnis\Job\test\transloco1\node_modules\@ngneat\transloco-keys-manager\helpers\getScopeAndLangFromFullPath.js:1
(function (exports, require, module, __filename, __dirname) { "use strict";function getScopeAndLangFromFullPath(e,t){!1===t.endsWith("/")&&(t=`${t}/`);const[n,o]=e.split(t),l=o.split("/");let p,r;return l.length>1?(r=l.pop().replace(".json",""),p=l.join("/")):r=l[0].replace(".json",""),{scope:p,lang:r}}Object.defineProperty(exports,"__esModule",{value:!0}),exports.getScopeAndLangFromFullPath=getScopeAndLangFromFullPath;

                          ^

TypeError: Cannot read property 'split' of undefined
    at Object.getScopeAndLangFromFullPath (C:\Users\Arnis\Job\test\transloco1\node_modules\@ngneat\transloco-keys-manager\helpers\getScopeAndLangFromFullPath.js:1:178)
    at Object.compareKeysToFiles (C:\Users\Arnis\Job\test\transloco1\node_modules\@ngneat\transloco-keys-manager\keysDetective\compareKeysToFiles.js:1:1086)
    at Object.findMissingKeys (C:\Users\Arnis\Job\test\transloco1\node_modules\@ngneat\transloco-keys-manager\keysDetective.js:1:897)
    at Object.<anonymous> (C:\Users\Arnis\Job\test\transloco1\node_modules\@ngneat\transloco-keys-manager\index.js:2:744)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! transloco1@0.0.0 i18n:find: `transloco-keys-manager find`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the transloco1@0.0.0 i18n:find script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/src/helpers/getScopeAndLangFromFullPath.ts
+++ b/src/helpers/getScopeAndLangFromFullPath.ts
@@ -2,7 +2,12 @@
  * /Users/username/www/folderName/src/assets/i18n/admin/es.json => { scope: admin, lang: es }
  * /Users/username/www/folderName/src/assets/i18n/es.json => { scope: undefined, lang: es }
  */
+import {toUnixFormat} from './toUnixFormat';
+
 export function getScopeAndLangFromFullPath(filePath: string, translationPath: string) {
+  filePath = toUnixFormat(filePath);
+  translationPath = toUnixFormat(translationPath);
+
   if (translationPath.endsWith('/') === false) {
     translationPath = `${translationPath}/`;
   }

--- a/src/helpers/toUnixFormat.ts
+++ b/src/helpers/toUnixFormat.ts
@@ -1,0 +1,5 @@
+import * as path from 'path';
+
+export function toUnixFormat(p) {
+  return p.split(path.sep).join('/');
+}

--- a/src/keysDetective/compareKeysToFiles.ts
+++ b/src/keysDetective/compareKeysToFiles.ts
@@ -41,6 +41,7 @@ export function compareKeysToFiles({ scopeToKeys, translationPath, addMissingKey
   }
 
   for (const file of translationFiles) {
+
     const { scope } = getScopeAndLangFromFullPath(file, translationPath);
     const keys = scope ? scopeToKeys[scope] : scopeToKeys.__global;
     if (keys) {

